website/content/en/docs/faq.md

```diff
@@ -123,10 +123,30 @@ func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Res
 	}
 ```
 
+## I see deepcopy errors and image build fails. How do I fix this?
+
+When you run the ```operator-sdk generate k8s``` command, you might see an error like this
+
+``` bash
+INFO[0000] Running deepcopy code-generation for Custom Resource group versions: [cache:[v1alpha1], ] 
+F0523 01:18:27.122034    5157 deepcopy.go:885] Hit an unsupported type invalid type for invalid type, from github.com/example-inc/memcached-operator/pkg/apis/cache/v1alpha1.Memcached
+```
+
+This is because of the ```GOROOT``` environment variable not being set. More details [here][goroot-github-issue].
+
+In order to fix this, you simply need to export the ```GOROOT``` environment variable
+
+``` bash
+export GOROOT=$(go env GOROOT)
+```
+
+This will work for the current terminal instance, to make it more permanent, consider adding this line to the ```bashrc``` file.
```

Comment:
```suggestion
This will work for the current environment. To persist this fix, add the above line to your environment's config file, ex. `.bashrc`.
```

Reply:
done